scripts/release/bump

```diff
@@ -103,7 +103,7 @@ fi
 
 bump
 
-if git status -s | grep -vq 'CHANGELOG.md\|cli/src/semgrep/__init__.py\|setup.py\|semgrep-core/src/cli/version.ml'; then
+if git status -s | grep -vq 'cli/src/semgrep/__init__.py\|cli/setup.py\|setup.py\|semgrep-core/src/cli-lib/version.ml'; then
```

Comment:
Do we need to remove `CHANGELOG.md` here?

Reply:
the changelog isn't changed until a later step in the workflow, so I don't think we need to keep this in there anymore.

Reply:
Plus, I think we want to make sure the changelog isn't changing prior to towncrier running.